bindings/rascal/representations/spherical_covariants.py

```diff
@@ -111,23 +108,16 @@ class documentation
                                     radial_contribution=radial_contribution)
 
         self.nl_options = [
-            dict(name='centers', args=[]),
-            dict(name='neighbourlist', args=[interaction_cutoff]),
-            dict(name='strict', args=[interaction_cutoff])
+            dict(name='centers', args=dict()),
```

Comment:
I really don't like this initialization style and performance are not an issue at that level (0.3 micro sec)

Reply:
It's silly to keep arguing about this. There are arguments for both initialization styles (e.g. the `dict()` style is more verbose, but it does tell you explicitly that you're initializing a dictionary rather than the similar-looking set object), though it does seem that the curly-brace style is more common in practice.

Just keep in mind that, if you ever need to use key names that aren't valid Python identifiers (e.g. spaces, dashes/hyphens/minus signs, starting with numbers, funky Unicode characters...) then you'll have to change it to the curly-brace style, and it's quite possible we'll want to change all similar uses for (internal) consistency.